initializr-service/src/main/resources/application.yml

```diff
@@ -1252,6 +1252,33 @@ initializr:
           id: remote-shell
           description: CRaSH shell integration
           versionRange: "[1.0.0.RELEASE,1.5.0.RC1)"
+        - name: Codecentric Spring Boot Admin Server
+          description: Provides the Spring Boot Admin Server API (which can be paired with the UI)
+          id: spring-boot-admin-server
+          artifactId: spring-boot-admin-server
+          version: 1.5.6
+          groupId: de.codecentric
+          links:
+            - rel: reference
+              href: http://codecentric.github.io/spring-boot-admin/1.5.6/#getting-started
```

Comment:
That version is going  to be  annoying to upgrade. A reference to `current` would be interesting.

Reply:
yeah you're right. looking into making this possible